docs/index.md

```diff
@@ -24,7 +24,7 @@ terraform {
 }
 
 provider "routeros" {
-  hosturl        = "https://router.local"        # env ROS_HOSTURL or MIKROTIK_HOST
+  hosturl        = "apis://router.local"         # env ROS_HOSTURL or MIKROTIK_HOST
```

Comment:
The documentation is auto-generated. Please make additions to https://github.com/terraform-routeros/terraform-provider-routeros/blob/main/examples/provider/provider.tf instead of this file.
And please do not remove the existing parameter, but add your own under the comment # so that the alternative is visible.  A lot of people manage routers via https.

Reply:
Sure thing, I should have realized that came from the code comments...

Reply:
I've left this one alone in the revision. I don't think it is confusing since the provider docs now call out the options (and it'd be good to guide toward using the new rest api by default).